README.md

```diff
@@ -162,14 +162,17 @@ Worklet script (i.e. `reach.js`):
 ```js
 class SendReachReportOperation {
   async function run(data) {
-    // A toy model that only computes reach for users whose favorite color is red.
-    if (data["favorite-color"] != "red") {
+    // A toy model that only computes reach for users who haven't previously had a report sent.
+    if (await this.sharedStorage.get("report-sent") != "yes") {
```

Comment:
I'm wondering if this "report sent" value should be scoped per campaign to measure per-campaign reach. (We could also consider fixing the bucket below instead so that this measures overall reach for all campaigns.)

Reply:
Never mind, I just reread what you wrote, and the second part makes more sense now.  :)